src/librustc/ich/impls_hir.rs

```diff
@@ -653,7 +653,7 @@ impl<'a> HashStable<StableHashingContext<'a>> for ast::Ident {
                                           hasher: &mut StableHasher<W>) {
         let ast::Ident {
             ref name,
-            ctxt: _ // Ignore this
+            span: _ // Ignore this
```

Comment:
@michaelwoerister 
Now I'm pretty sure this is a mistake (and ignoring `ctxt` was a mistake as well?).
Spans are hashed in other positions, including the `ctxt` part.

Reply:
Yes, that should be updated! Good catch.